include/groonga/util.h

```diff
@@ -23,6 +23,30 @@
 extern "C" {
 #endif
 
+/**
+ * \brief Inspect specified object and produce a textual representation.
+ *
+ * \since 0.7.0
```

Comment:
```suggestion
```

In my opinion, this is needless.
Because we can use this function since almost initial release.

Reply:
fix: [c15989b](https://github.com/groonga/groonga/pull/2328/commits/c15989bf741e91ee61795156b24d918beb8b7ae4)